README.md

```diff
@@ -1,13 +1,20 @@
-# jkennedy-research-and-reflection
+# Jacob Kennedy - Research & Reflection
 
+## Course goals
```

Comment:
As above. As this files grows you'll need to create clear sections to find content.